<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>boundary</class>
 <widget class="QFrame" name="boundary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="Lbl_H1">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;BOUNDARY CONDITIONS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_Notice">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Boundary conditions are already prescribed through the shallow-water solutions&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="SWg_Interface">
     <property name="toolTip">
      <string>Interface points for all SW-CFD interfaces</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLY_AddBuild">
     <item>
      <widget class="QToolButton" name="Btn_AddBoundary">
       <property name="toolTip">
        <string>Add building data</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <property name="text">
        <string>Add Boundary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="Btn_RemBoundary">
       <property name="toolTip">
        <string>Remove building data</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <property name="text">
        <string>Remove Boundary</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLY_Scroller" stretch="0,0,0,0">
     <item>
      <spacer name="HSp_Scroll01">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Previous">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-image: url(:/images/images/previous.png) 10;
    border-width: 0px;                         
    padding: 0px 0px;
    min-height: 10px;
    min-width: 10px;
}

QPushButton:hover {
    border-image: url(:/images/images/previous.png) 10;
}

QPushButton:pressed {
    color: lightgray;
    border-image: url(:/images/images/previous-invert.png) 10;
    padding-top: 0px;
    padding-bottom: 0px;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Next">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-image: url(:/images/images/next.png) 10;
    border-width: 0px;                         
    padding: 0px 0px;
    min-height: 10px;
    min-width: 10px;
}

QPushButton:hover {
    border-image: url(:/images/images/next.png) 10;
}

QPushButton:pressed {
    color: lightgray;
    border-image: url(:/images/images/next-invert.png) 10;
    padding-top: 0px;
    padding-bottom: 0px;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="HSp_Scroll02">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="VSp">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>363</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
